### ISSUE TITLE
Model Training and Validation matrix Calculation

### ISSUE THREAD

**Issue by Prasham2181** [shivamshinde123/Instrument-Based-Music-Recommendation#9]

**Comment by Prasham2181** [comment]
Model trained and weighted_MSE loss = 0.0022096074838191273 achived
